core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -559,7 +559,7 @@ struct SubviewExtents {
   // std::pair range
   template <size_t... DimArgs, class... Args>
   void error(char* buf, int buf_len, unsigned domain_rank, unsigned range_rank,
-             const ViewDimension<DimArgs...>& dim, const Kokkos::Impl::ALL_t,
+             const ViewDimension<DimArgs...>& dim, const ALL_t,
```

Comment:
do we even need the `const` before ALL_t here?

Reply:
No.  Want me to remove?

Reply:
i would say yes for consistency with the other args but no big deal

Reply:
Done